doc/content/integrations/cloud-integrations/aws-iot/default/deployment-guide.md

```diff
@@ -64,6 +66,9 @@ The parameters configure the integration:
 
 - **Principal Account ID** {{< distributions "Enterprise" >}}: AWS Account ID that The Things Stack authenticates with.
 - **Thing Type Name**: The unique AWS IoT Core thing type name for this integration.
+- **Thing Name Scheme**: The name that is given to AWS IoT things when they are created by the integration.
+  - When using DevEUI, the thing name will be like 1122334455667788.
+  - When using DeviceID, the thing name will be a combination of the CloudFormation stack name and the device ID as registered in  {{% tts %}}. It would be like `<stack-name>_<device-id>`.
```

Comment:
```suggestion
  - When using a DevEUI, the thing name will be like `1122334455667788`.
  - When using a DeviceID, the thing name will be a combination of the CloudFormation stack name and the device ID as registered in  {{% tts %}}, i.e. `<stack-name>_<device-id>`.
```

Reply:
It should be "When using 'DevEUI/DeviceID'", because that's the name of the scheme.